src/foam/u2/view/ColumnConfigView.js

```diff
@@ -536,10 +541,22 @@ foam.CLASS({
     display: flex;
     align-items: center;
     justify-content: start;
+    width: 100%;
   }
   ^selection-buttons + ^selection-buttons {
     padding: 8px;
   }
+  ^labelText {
+    flex: 1;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+  ^labelText:hover {
```

Comment:
Not sure this is the best idea anymore, Would cause unexpected UI changes. We can remove for now and look into a better solution

Reply:
hm ok